src/mman-win32.mk

```diff
@@ -7,7 +7,7 @@ $(PKG)_VERSION  := 3421c28e753c38d24a2e27c111b1c9b4601ebe7d
 $(PKG)_CHECKSUM := c33e84043d49d0e33bc434bda3a16ce60432e789
 $(PKG)_SUBDIR   := mman-win32-master
 $(PKG)_FILE     := mman-win32-$($(PKG)_VERSION).tar.gz
-$(PKG)_URL      := https://github.com/witwall/mman-win32/archive/master.tar.gz
+$(PKG)_URL      := https://github.com/witwall/mman-win32/archive/3421c28e753c38d24a2e27c111b1c9b4601ebe7d.zip
 $(PKG)_DEPS     := gcc
 
 define $(PKG)_UPDATE
```

Comment:
You might find `MXE_GET_GITHUB_SHA` useful. See how it's used: https://github.com/mxe/mxe/blob/4a758acf46f140ad50fb1114d16dfc01f313cd1b/src/librtmp.mk#L13